src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -316,7 +288,156 @@ To ensure your CSS files are accessible, add the `crossorigin="anonymous"` attri
+                          text
+                          textArea
+                          time
+                          url
+                          week
+                        }
+                      }
+                    }
+                    errors {
+                      field
+                      errorClass
+                      description
+                    }
+                  }
+                }
+                ```
+            </td>
+            <td>
+                ```json
+                {
+                  "sessionReplay": {
+                    "enabled": true,
+                    "collectFonts": true,
+                    "blockSelector": "String",
+                    "maskTextSelector": "String",
+                    "inlineStylesheet": false,
+                    "maskAllInputs": true,
+                    "maskInputOptions": {
+                      "color": true,
+                      "date": true,
+                      "textArea": true
+                    },
+                    "samplingRate": 9.99,
+                    "errorSamplingRate": 1.11
+                  }
+                }
+                ```
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Data consumption [#data-consumption]
+
+### View consumption [#view-consumption]
+
+Between April 8 and May 8, 2024, while session replay bytes are free, you can view your consumption to help with planning once the bytes become paid:
+    1. Go to <DoNotTranslate>**Application Settings**</DoNotTranslate>.
+    2. Scroll down to <DoNotTranslate>**Session replay**</DoNotTranslate>.
+    3. In the <DoNotTranslate>**Session sample size**</DoNotTranslate> section, click <DoNotTranslate>**View your account's consumption**</DoNotTranslate>.
+
+    <img
+        title="View account consumption on application settings page"
+        alt="screenshot of application settings page"
+        src={browserViewConsumptionDetails}
+    />
+
+### Plan for paid consumption [#plan-for-paid-consumption]
+
+After May 8, 2024, session replay will follow the same consumption pricing as your other browser bytes. The amount of bytes produced depends on the count, length, and user-activity levels of sessions, as well as the complexity of your site's DOM.
+
+The most accurate way to project your cost per replay is to enable the feature for a short test period and measure your actual consumption. Alternatively, you can use an average of 5.3 MB per replay as a rough approximation, though keep in mind that your actual cost may vary significantly. A simple formula to project your approximate costs is: `sessions` x `sampling_rate` x `gb_per_replay` x `cost_per_gb`
+        * Example: `sessions: 1 million` x `sampling_rate: 5%` x `gb_per_replay: 0.0053` x `cost_per_gb: $0.30` = `$79.50 for 50,000 replays`
+
+To control your consumption, [adjust your sampling rates](#configure-sampling-rates).
+    
+## User privacy and security concerns [#data-security]
```

Comment:
I understand not wanting to bulk up the steps section of the doc, but this seems like super useful/important context for the configure privacy settings step.